QRCoder/QRCodeGenerator.cs

```diff
@@ -1107,11 +1107,7 @@ private static string ConvertToIso8859(string value, string Iso = "ISO-8859-2")
             Encoding utf8 = Encoding.UTF8;
             byte[] utfBytes = utf8.GetBytes(value);
             byte[] isoBytes = Encoding.Convert(utf8, iso, utfBytes);
-#if NETFRAMEWORK || NETSTANDARD2_0 || NET5_0
             return iso.GetString(isoBytes);
```

Comment:
Before your simplified to `iso.GetString(isoBytes, 0, isoBytes.Length)` (the one with three overloads. See: https://github.com/codebude/QRCoder/pull/503/files#diff-6752f108aa953a2872b67db5e51503d099b3f0ddb0ae668433a2ba73e12eb13aL2522

I would say this should be done here, too, because when I implemented it, with .NET Core there was no GetString() overload with just one parameter. On the other hand, it seems like the test cases passed successfully - but how?

Reply:
So, you are suggesting that I change it to the one with three parameters?  Do you think the one with three parameters is better?  Or just for consistency?  Maybe we should change the other one to have just a single parameter, for simplicity?

According to the .NET documentation, the single-parameter version is supported by .NET Framework 1.1+, .NET Standard 1.3+, and all versions of .NET Core.  Perhaps you had previously targeted an earlier version of .NET Standard.  But now QRCoder's base .NET Standard target is 1.3.

https://learn.microsoft.com/en-us/dotnet/api/system.text.encoding.getstring?view=net-8.0#system-text-encoding-getstring(system-byte())

Reply:
Unless you specify otherwise, I changed the other reference to use the overload with a single parameter.

Reply:
Ok, then maybe the signatures have changed over time. Or I initially targeted another .net core version. Current solution is fine for me.